plugin/src/main/java/org/opensearch/ml/settings/MLCommonsSettings.java

```diff
@@ -117,6 +117,27 @@ private MLCommonsSettings() {}
     public static final Setting<Boolean> ML_COMMONS_REMOTE_INFERENCE_ENABLED = Setting
         .boolSetting("plugins.ml_commons.remote_inference.enabled", true, Setting.Property.NodeScope, Setting.Property.Dynamic);
 
+    public static final Setting<Boolean> ML_COMMONS_REMOTE_INFERENCE_RETRY_ENABLED = Setting
+        .boolSetting("plugins.ml_commons.remote_inference.retry_enabled", true, Setting.Property.NodeScope, Setting.Property.Dynamic);
+
+    public static final Setting<Integer> ML_COMMONS_REMOTE_INFERENCE_RETRY_BACKOFF_MILLIS = Setting
+        .intSetting(
+            "plugins.ml_commons.remote_inference.retry_backoff_millis",
+            100,
+            1,
```

Comment:
Why skipping the max value? I think it makes sense to put a max allowed value here. Same for ML_COMMONS_REMOTE_INFERENCE_RETRY_TIMEOUT_SECONDS

Reply:
ack, will set to 5s and 60s